dist/configs/config/server.cfg

```diff
@@ -58,7 +58,7 @@ set timelimit 0
 set g_emptyTeamsSkipMapTime 15
 
 // force each team to have a similar number of players
-set g_teamForceBalance 1
+set g_teamForceBalance 2
```

Comment:
Wording suggestion:

```c
// team enforcement
// 0 - no restriction
// 1 - force each team to have a similar number of players
// 2 - force each team to have a similar number of players once there are players in both team, make "players versus bots" possible if no player joins the other team
set g_teamForceBalance 2
```

Reply:
I've pushed another formulation, that is
```diff
 // 1 - force each team to have a similar number of players
+// 2 - if players start going in both teams, force each team to have a similar number of players; otherwise players can all join the same team for a human vs bots game
 set g_teamForceBalance 2
```

Tell me which you prefer